src/components/editor/EditAuthorsForm.jsx

```diff
@@ -60,19 +62,22 @@ export default class EditAuthorList extends React.Component {
     let newValue = {};
     newValue[slug] = false;
     let newAuthorsDeleted = Object.assign({}, this.state.authorsDeleted, newValue);
-    this.setState({
+    this.safeSetState({
       authorsDeleted: newAuthorsDeleted
     });
   }
 
+  handleStopPropagation(event) {
+    event.stopPropagation();
+  }
+
   render() {
     return (
       <div>
-        Add Author:<br/>
-        <input placeholder="Input Slug" name="addAuthor" ref={(ref) => {this.addAuthorNode = ref}} />
-        <button type="button" onClick={this.handleAddAuthor}>Add Author</button>
+        <input placeholder="Add Author" onChange={this.handleStopPropagation} ref={(ref) => {this.addAuthorNode = ref}} />
```

Comment:
Just use a lambda expression to stop the propagation.

Also yeah I decided I don't like these refs.